astropy/convolution/tests/test_convolve_models.py

```diff
@@ -0,0 +1,94 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import math
+import numpy as np
+from ..convolve import convolve, convolve_models
+from ...modeling import models, fitting
+from ...tests.helper import pytest
+from ...utils.misc import NumpyRNGContext
+from numpy.testing import assert_allclose, assert_almost_equal
+
+try:
+    import scipy
+except ImportError:
+    HAS_SCIPY = False
+else:
+    HAS_SCIPY = True
+
+class TestConvolve1DModels(object):
+    @pytest.mark.parametrize('mode', ['convolve_fft', 'convolve'])
+    @pytest.mark.skipif('not HAS_SCIPY')
+    def test_against_scipy(self, mode):
+        from scipy.signal import fftconvolve
+
+        kernel = models.Gaussian1D(1, 0, 1)
+        model = models.Gaussian1D(1, 0, 1)
+        model_conv = convolve_models(model, kernel, mode=mode)
+        x = np.arange(-5, 6)
+        ans = fftconvolve(kernel(x), model(x), mode='same')
+
+        assert_allclose(ans, model_conv(x) * kernel(x).sum(), atol=1e-5)
+
+    @pytest.mark.parametrize('mode', ['convolve_fft', 'convolve'])
+    @pytest.mark.skipif('not HAS_SCIPY')
+    def test_against_scipy_with_additional_keywords(self, mode):
+        from scipy.signal import fftconvolve
+
+        kernel = models.Gaussian1D(1, 0, 1)
+        model = models.Gaussian1D(1, 0, 1)
+        model_conv = convolve_models(model, kernel, mode=mode,
+                                     normalize_kernel=False)
+        x = np.arange(-5, 6)
+        ans = fftconvolve(kernel(x), model(x), mode='same')
+
+        assert_allclose(ans, model_conv(x), atol=1e-5)
+
+    @pytest.mark.parametrize('mode', ['convolve_fft', 'convolve'])
+    def test_sum_of_gaussians(self, mode):
+        """
+        Test that convolving N(a, b) with N(c, d) gives N(a + b, c + d),
```

Comment:
I think this is wrong, but the test isn't:
`convolution(N(a,b), N(c,d)) = N(a+c, (b^2+d^2)^0.5)`
that's implemented in the test, so just fix the docs